user-manuals/orders/bopis/order-details-page/README.md

```diff
@@ -1,42 +1,31 @@
 # Order Details Page
 
-Access the Order Details page by clicking on an order card, it will display all order-related information such as customer name, order id, customer's email address, item details, reason for rejection and order item rejection history.
+Users can access the Order Details page by clicking on any order card from any of the tabs on Order page, it will display all order-related information for that order such as customer’s name, order ID, customer's contact details, item details, reason for rejection and order item rejection history.
 
-Access the open order page, select an order card. Users will then be directed to the order details page. The order details page enables store associates to perform the following functions:
+On clicking on any order card in the Open or Packed tab, users will be directed to the order details page. The order details page displays key details of an order and also enables store associates to perform the certain functions. 
 
-**Reject Orders**
+### Reject Orders
 
-During peak hours or promotional events, brick-and-mortar retail locations often face increased in-store traffic, heightening the risk that the requested item for pick-up may become unavailable due to in-store purchases. In such instances, store associates may need to reject the order or reject partial order in case the order item inventory is insufficient at the store. This feature can be enabled using the toggle on the settings page.
+During peak hours or promotional events, brick-and-mortar retail locations often face increased in-store traffic, heightening the risk that the requested item for pick-up may become unavailable due to in-store purchases. In such instances, store associates may need to reject an order or reject an order partially in case the order item inventory is insufficient at the store. Partial order rejection feature can be enabled using the toggle on the settings page.
 
-To reject an order, access the open order page. Select the order card to navigate to the order details page. Choose reasons for rejection. Click on the 'Report an Issue' button to access a pop-up menu bar for selecting rejection reasons. Confirm the rejection by clicking on the save icon.
+To reject an order, click on the order card in the `Open` tab to open the order details page. Click on `Reject Order` to reject the order, select an appropriate reason for rejecting the order in the pop-up window that appears and click on the `save` icon. Another pop-up will appear to confirm order rejection, select `Reject` to reject the order. 
```

Comment:
also need to include how partial orders are rejected

Reply:
Added.